src/main/resources/sql/init.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE groups
+(
+    id   BIGINT PRIMARY KEY NOT NULL,
+    name TEXT NOT NULL UNIQUE
+);
+
+CREATE TABLE students
+(
+    id            BIGINT PRIMARY KEY NOT NULL,
+    name          TEXT      NOT NULL,
+    surname       TEXT      NOT NULL,
+    date_of_birth TIMESTAMP NOT NULL,
+    CONSTRAINT fk_group FOREIGN KEY (group_id) REFERENCES groups (id)
```

Comment:
Also we need to add column `group_id`, because this constraction `CONSTRAINT fk_group FOREIGN KEY (group_id) REFERENCES groups (id)` will not work without column `group_id`.